src/search/engines/startpage/options.go

```diff
@@ -5,7 +5,7 @@ import "github.com/hearchco/hearchco/src/search/engines"
 var Info = engines.Info{
 	Domain:         "www.startpage.com",
 	Name:           engines.STARTPAGE,
-	URL:            "https://www.startpage.com/sp/search?q=",
+	URL:            "https://www.startpage.com/do/search?language=english&q=",
```

Comment:
What does `/do` do? As I can see, it's getting redirected to `/sp` anyways?

Reply:
Its what startpage sent me to when requesting i just copied and paste. Didnt notice it tho